src/Pages/ProductList/ProductList.component.js

```diff
@@ -3,15 +3,87 @@ import React, { Component } from "react";
 import "./ProductList.styles.scss";
 import ProductCard from "./ProductCard/ProductCard.component";
 
+const API = "http://localhost:3000/data/productlist/productlist.json";
+// const API = "http://10.58.4.20:8000/category?main=1"; //Backend API
+
 class ProductList extends Component {
+  state = {
+    isCategoryClick: false,
+    isSortingClick: false,
+    categories: "",
+    products: "",
+  };
+
+  handleSorting = () => {
+    const { isSortingClick } = this.state;
+    this.setState({
+      isSortingClick: !isSortingClick,
+    });
+  };
+
+  componentDidMount = async () => {
+    const response = await fetch(API);
+    // custom error handler
+    if (response.status !== 200) {
+      throw new Error("cannot fetch the data");
+    }
+    const data = await response.json();
+    this.setState({ categories: data.categories, products: data.products });
+  };
+
   render() {
+    const { isSortingClick, categories, products } = this.state;
     return (
       <section className="ProductList">
-        <div className="row">
-          <div className="col s12 m4">
-            <ProductCard />
+        <div className="category-nav">
+          <div className="category-title">
+            <i className="far fa-carrot"></i>
```

Comment:
```suggestion
            <i className="far fa-carrot" />
```
셀프 클로징 태그 적용할 수 있습니다

Reply:
적용 완료!! 🤟🏻🤟🏻🤟🏻🤟🏻